src/class-list.ts

```diff
@@ -0,0 +1,34 @@
+function _classesOf(element): string[] {
+  return element.className.trim().split(/\s+/);
+}
+
+export function add(element, ...args): void {
+  const toAdd = Array.from(args);
+  const className = _classesOf(element)
```

Comment:
this variable is redundant, i'd rather see this written as `element.className` =

Reply:
:+1: